src/components/colorPicker/index.tsx

```diff
@@ -65,15 +66,23 @@ const ACCESSIBILITY_LABELS = {
 class ColorPicker extends PureComponent<Props> {
   static displayName = 'ColorPicker';
 
-  static defaultProps = {
-    accessibilityLabels: ACCESSIBILITY_LABELS,
-    backgroundColor: Colors.$backgroundDefault
-  };
+  constructor(props: Props) {
+    super(props);
+
+    if (props.style) {
+      LogService.warn('UILib ColorPicker style prop is deprecated.');
```

Comment:
```suggestion
      LogService.warn('UILib ColorPicker style prop is deprecated. Please stop using it.');
```

Reply:
You should also tell the user what to do, otherwise, they will ask if there is another prop that replaces it etc.